plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -36,29 +37,33 @@ type Props = {
 
 export const AboutContent = ({ entity }: Props) => {
   const classes = useStyles();
+  const [partOfSystemRelation] = getEntityRelations(entity, RELATION_PART_OF, {
+    kind: 'system',
+  });
+  const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY);
+
   return (
     <Grid container>
       <AboutField label="Description" gridSizes={{ xs: 12 }}>
         <Typography variant="body2" paragraph className={classes.description}>
           {entity?.metadata?.description || 'No description'}
         </Typography>
       </AboutField>
+      <AboutField label="Owner" gridSizes={{ xs: 12, sm: 6, lg: 4 }}>
+        {ownedByRelations.map((t, i) => [
```

Comment:
I think it would be better to return a fragment with a key here right? Afaik this will disable any array optimizations in React